### PR TITLE
Post watch

### PR DESCRIPTION
**Pull request by ckim0419**
Here goes nothing.

### CODE REVIEW

**Comment by ckim0419** [comment]
Maybe I should make psutil a requirement.

**Comment by dschep** [comment]
Yup, I'd suggest making it an extra (see xmpp in `setup.py`).

**Comment by dschep** [comment]
Oh, you did. Probably needs to be in `tests_require`.

File: ntfy/cli.py [hunk 1]
```diff
@@ -32,6 +37,28 @@ def run_cmd(args):
...
+        exit(1)
+    try:
+        p = psutil.Process(args.pid)
+        cmd = p.cmdline()
+    except psutil.NoSuchProcess:
+        logging.error("PID {} not found".format(args.pid))
+        exit(1)
+    p.wait()
+    if emojize is not None and not args.no_emoji:
+        prefix = '\u2705 ' if retcode == 0 else '\u274C '
```
Comment (dschep) [inline ntfy/cli.py]: Mind changing these to :white_check_mark: and :x: similar to 9f85fd.

File: ntfy/cli.py [hunk 2]
```diff
@@ -32,6 +37,28 @@ def run_cmd(args):
...
+    else:
+        prefix = ''
+    return '{}"Process[{}]: {}" was finished at {}'.format(
```
Comment (dschep) [inline ntfy/cli.py]: Can you not find out the start time and compute run time? Most notifications include the date already :wink:

**Comment by dschep** [comment]
Dig it! Any chance this could be `ntfy done --pid XXXX` instead of `ntfy watch XXXX` or is that a limitation of using `REMAINDER` in the `done_parser`?

**Pull request closed without merging**